packages/map-template/src/components/Map/Map.jsx

```diff
@@ -80,16 +87,31 @@ function Map({ apiKey, gmApiKey, mapboxAccessToken, venues, venueName, onLocatio
         return mapsIndoorsInstance.fitVenue(venue);
     }
 
+    /**
+     * Replace the default tile URL style to the incoming tile style.
+     */
+    const onTileStyleChanged = (miInstance) => {
```

Comment:
Can we "exit early" if there is no tileStyle prop set at all? No need to do unnecessary work.

Reply:
Would a check if `_tileStyle` exists be enough? We could add a check next to the `miInstance`, and then it would only run if we have tile styles.

Reply:
Give it a go 👍